scilpy/io/utils.py

```diff
@@ -146,12 +151,16 @@ def validate_sh_basis_choice(sh_basis):
 
 
 def assert_inputs_exist(parser, required, optional=None):
-    """
-    Assert that all inputs exist. If not, print parser's usage and exit.
-    :param parser: argparse.ArgumentParser object
-    :param required: string or list of paths
-    :param optional: string or list of paths.
-                     Each element will be ignored if None
+    """Assert that all inputs exist. If not, print parser's usage and exit.
+
+    Parameters
+    ----------
+    parser: argparse.ArgumentParser object
+        Parser.
+    required: string or list of paths
+        Paths to be checked.
```

Comment:
Modify the doc please to be more explicit. For example: `Requiered paths to be checked`.

Reply:
I would then change the variable name to a more meaningful name, since `required` along is not meaningful to me. Wouldn't `required_paths` be more appropriate?

Reply:
We are going to keep these var names. Please update the doc ;)

Reply:
Done in 4a5df3b.